Components/input.css

```diff
@@ -0,0 +1,40 @@
+.input {
+  width: 100%;
+  margin: 0.5rem;
+}
+
+.input input {
+  padding: 0.45rem;
+  outline: none;
+  background-color: var(--blueblack-100);
+  border-radius: 5px;
+  margin-top: 5px;
```

Comment:
recommend to use relative units.

Reply:
Don't use px it won't be responsive, instead use rem or em.